### ISSUE TITLE
Nodemailer and sendmail

### ISSUE THREAD

**Issue by tprinslo** [Ylianst/MeshCentral#3299]
Hi MeshCentral team,
I see that Nodemailer allows the usage of the sendmail command (  https://nodemailer.com/transports/sendmail ).
Will it be possible to add support for the usage of sendmail with Nodemailer in Meshcentral?

**Comment by Ylianst** [comment]
Good request, just added support for it in upcoming MeshCentral v0.9.53. You can add this section to the config.json:

```
  "sendmail": {
    "newline": "unix",
    "path": "/usr/sbin/sendmail",
    "args": [ "-f", "foo@example.com" ]
  }
```

If you want to keep all the values to defaults, just add this:

```
  "sendmail": { }
```

Once v0.9.53 is published, let me know if that works.

**Comment by Ylianst** [comment]
![MC2-Sendmail](https://user-images.githubusercontent.com/1319013/143493653-4c26cb7a-cf6f-4720-8a33-86b6d18a0084.png)

**Comment by Ylianst** [comment]
v0.9.53 is out :)

**Comment by tprinslo** [comment]
Hi Ylianst
Thanks for the quick response.
I made the changes in the config.json but it is not working in my environment. 

When it start Meshcentral manually I do not see any messages regarding email services successfully started as with SMTP
`node ./node_modules/meshcentral
MeshCentral HTTP redirection server running on port 80.
MeshCentral v0.9.53, WAN mode.
MeshCentral Intel(R) AMT server running on meshdemo.ncc.co.za:4433.
MeshCentral HTTPS server running on meshdemo.ncc.co.za:443.`

In the “my server console” it confirms that the email service is not running when using the email command
`  > email
No email service enabled.`

Running the following test program using nodemailer and sendmail on the same server works fine
`var nodemailer = require('nodemailer');
let transporter = nodemailer.createTransport({
    sendmail: true,
    newline: 'unix',
    path: '/usr/sbin/sendmail'
});
transporter.sendMail({
    from: 'noreply@meshdemo……..',
    to: '…..@.....',
    subject: 'Message from Meshdemo',
    text: 'I hope this message gets delivered!'
}, (err, info) => {
    console.log(info.envelope);
    console.log(info.messageId);
});  `

**Comment by Ylianst** [comment]
Ok, checking again... I probably messed up something.

**Comment by Ylianst** [comment]
My fault, just put in a fix for this. Will be with v0.9.54 later today.

**Comment by tprinslo** [comment]
Super Thanks

**Comment by Ylianst** [comment]
Note that I don't expect any confirmation when the server starts that the SMTP server is correct. "sendmail" is a command, I don't think there is a way to test the connection in this mode. Regardless, with the new version, "email" command in the server console will work.

**Comment by tprinslo** [comment]
Noted thanks

**Comment by tprinslo** [comment]
Hi Ylianst
Tested the sendmail option and it works great!!

**Comment by Ylianst** [comment]
Nice. Thanks for reporting back. I did make a few more fixed that will be in v0.9.55 since the module "nodemailer" was not being auto-installed when using "sendmail" and fixed per-domain configuration of sendmail. Should not impact you.
